ipywidgets/widgets/tests/test_traits.py

```diff
@@ -19,3 +21,19 @@ class TestColor(TraitTestBase):
     _good_values = ["blue", "#AA0", "#FFFFFF"]
     _bad_values = ["vanilla", "blues"]
 
+
+class TestBuffers(TestCase):
+    def test_state_with_buffers(self):
+        mv1 =  memoryview(b'test1')
+        mv2 =  memoryview(b'test2')
+        state = {'plain': [0, 'text'], 'x': {'ar': mv1}, 'y': {'shape': (10, 10), 'data': mv1}, 'z': [mv1, mv2], 'top': mv1}
```

Comment:
Can we add one more item that is nested several levels deep? Perhaps something like:

`'nested': {'a': mv1, 'b': [mv2, {'c': mv1, 'd': [mv1, mv2]}]}`